packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -584,12 +585,6 @@ test.serial('replace operators', async t => {
     }
   }
 
-  if (defaultManagerType === 'xs-worker') {
-    // XXX for some reason the code after this when run under XS fails with:
-    // message: 'unsettled value for "kp2526"',
-    return;
-  }
-
```

Comment:
Was this intentional?

Reply:
Yes. This whole file is now skipped under xs since https://github.com/Agoric/agoric-sdk/commit/cab3fbe1f1f01c484da0a745830d2e85513c2d44